README.md

```diff
@@ -42,37 +79,19 @@ and import plotly.js as
 
 ```js
 import Plotly from 'plotly.js-dist-min'
-// Or using require,
-var Plotly = require('plotly.js-dist-min')
 ```
-
-### Use the plotly.js CDN hosted by Fastly
-
-#### A minified plotly.js X.Y.Z release
-```html
-<script src="https://cdn.plot.ly/plotly-1.58.4.min.js" charset="utf-8"></script>
-```
-
-#### An un-minified version is also available
-```html
-<script src="https://cdn.plot.ly/plotly-1.58.4.js" charset="utf-8"></script>
+Or
+```js
+var Plotly = require('plotly.js-dist-min')
 ```
 
-and use the `Plotly` object in the window scope.
-
-##### Please note that as of v2 the "plotly-latest" outputs (e.g. https://cdn.plot.ly/plotly-latest.min.js) will no longer be updated on the CDN, and will stay at the last v1 patch v1.58.4. Therefore, to use the CDN with plotly.js v2 and higher, you must specify an exact plotly.js version.
-
-Fastly supports Plotly.js with free CDN service. Read more at <https://www.fastly.com/open-source>
-
-### Download the latest release or a release candidate (rc)
-
-[Latest and rc releases on GitHub](https://github.com/plotly/plotly.js/releases/)
+---
+## Versioning
```

Comment:
This probably doesn't need to be a top-level item, just a little note somewhere, maybe under the NPM section. it's kind of the norm anyway.

Reply:
Done in https://github.com/plotly/plotly.js/pull/5705/commits/a0bd25835c3f415dece9c5cd8604663a60d7f532.